EasyPost.Tests/PickupTest.cs

```diff
@@ -73,8 +79,14 @@ public async Task TestCancel()
             //use "TestCreate"
             Pickup pickup = await CreateBasicPickup();
 
+            // Uncomment the following line if you need to re-record the cassette
+            // Thread.Sleep(10000); // Wait enough time to process
```

Comment:
No need to wait here. The label and scanform batch tests are the only ones that should need this